Gopkg.toml

```diff
@@ -66,9 +67,13 @@ required = [
   name = "github.com/evalphobia/logrus_sentry"
   version = "0.4.6"
 
+[[constraint]]
+  name = "github.com/prometheus/client_golang"
+  revision = "da11cf2c83c7414b76736025847c1cbdbb41e11e"
+
 [[constraint]]
   name = "github.com/fabric8-services/fabric8-common"
-  revision = "master"
+  revision = "45960af9689499e9f5e5a4ebd142a8189a19ca1e"
```

Comment:
Is there a huge with needed to align with latest? Is common planned to be version or we want to fail asap pointing to master?

Reply:
It could randomly break our build/CI because fabric8-common@master might have a breaking change which this service doesn't care about. Usually every version of fabric8-common is stable and usable, so we should be OK to point to a specific version and forget about it for a while :)